thanks/thanks.py

```diff
@@ -1,82 +1,97 @@
 # -*- coding: utf-8 -*-
 
+from collections import namedtuple
 from humanfriendly.tables import format_pretty_table
 import json
 import os
-try:
-    import pip._internal as pip
-except ImportError:
-    import pip
 import requirements
+import requests
 from termcolor import colored, cprint
-try:
-    import xmlrpclib
-except ImportError:
-    import xmlrpc.client as xmlrpclib
 
 
-json_file = ("{}/thanks.json".format(os.path.dirname(os.path.realpath(__file__))))
-client = xmlrpclib.ServerProxy('https://pypi.python.org/pypi')
+JSON_FILE = ("{}/thanks.json".format(os.path.dirname(os.path.realpath(__file__))))
+
+ProjectData = namedtuple('ProjectData', ['name', 'funding_link', 'authors'])
```

Comment:
Using a named tuple is a great idea!

Reply:
I'm always thankful to Raymond Hettinger for these. Which reminds me I've yet to watch his talk on then new DataClasses.